Automattic/wp-calypso
# Plans (State): Remove introductoryOffer-releated props/selectors from Redux site-plans

… Redux site-plans



Part of addressing https://github.com/Automattic/wp-calypso/issues/86638

## Proposed Changes

The following props/selectors are removed as unused (and redundant):

* `introductoryOfferFormattedPrice` and `introductoryOfferRawPrice` properties from the Redux `SitePlanData` object
* `isIntroductoryOfferAppliedToPlanPrice`

## Why are these changes being made?


We only need one form of handling plan pricing in the code base, more so one framework. Anything else risks having alternative and differing views of the same data propagating across the code base.

## Testing Instructions



* Build passing should be enough
* Confirm no references or mentions in untyped files (string search)

## Pre-merge Checklist
